src/main/java/com/ionic/sdk/agent/SdkVersion.java

```diff
@@ -41,8 +41,24 @@ private static String getVersionStringInternal() {
         return (version == null) ? VERSION : version;
     }
 
+    /**
+     * Define the SDK major version number.
+     */
+    public static final int IONIC_SDK_MAJOR = 0;
+
+    /**
+     * Define the SDK minor version number.
+     */
+    public static final int IONIC_SDK_MINOR = 0;
```

Comment:
Shouldn't this be set to 3?

Reply:
The content of this file is only used when the code is run outside of our JAR packaging (if someone compiled the source, for example).

We're talking about addressing this in a follow on release...